tools/testbench/topology.c

```diff
@@ -240,6 +240,12 @@ static int tplg_load_filewrite(int comp_id, int pipeline_id, int size,
 			return -EINVAL;
 		}
 
+		if (!is_valid_priv_size(total_array_size, size, array)) {
```

Comment:
Is there a config where the testbench can be built without the tplg_parser? If not then this is fine.

Reply:
topology.c file includes a header from tplg_parser ([#L18](https://github.com/thesofproject/sof/blob/master/tools/testbench/topology.c#L18)). So, for this to compile, it's necessary that it needs to be built with tplg_parser.